src/app/public/modules/list-view-checklist/list-view-checklist.component.ts

```diff
@@ -252,82 +259,6 @@ export class SkyListViewChecklistComponent extends ListViewComponent implements
     this.dispatcher.next(new ListSelectedSetItemsSelectedAction([item.id], true, true));
   }
 
-  public clearSelections() {
```

Comment:
These are public functions and thus would be breaking changes...... I think we have to leave them even if we mark them as deprecated.

Reply:
I guess you're right. I've put them back with deprecated decorators.

Reply:
I've also confirmed these methods still work properly, so if a consumer is still using them, they'll be OK.